include/llbuild/Basic/FileInfo.h

```diff
@@ -60,6 +60,24 @@ struct FileTimestamp {
   }
 };
 
+struct FileChecksum {
+  uint8_t bytes[32] = {0};
+
+  bool operator==(const FileChecksum& rhs) const {
+    for(int i=0; i<32; i++) {
```

Comment:
replacing this with memcmp may be more efficient, even if these comparisons aren't a huge bottleneck